### ISSUE TITLE
Addition of DiFuMo Atlases

### ISSUE THREAD

**Issue by FL33TW00D** [nilearn/nilearn#2422]
Hi there,
I have just read the new paper from Dadi et Al and would like to use the new DiFuMo Atlases, have they been added to Nilearn yet? 
If not I'd be happy to assist.

Thanks

**Comment by bthirion** [comment]
You can get them here:
https://parietal-inria.github.io/DiFuMo/notebook/index.html
Best,

**Comment by FL33TW00D** [comment]
Many thanks for this Bertrand

**Comment by jeromedockes** [comment]
I guess this can be closed?

**Comment by GaelVaroquaux** [comment]
> I guess this can be closed?

Let's keep this open: I think that it would be great to integrate the
code once the paper is published.

**Comment by bthirion** [comment]
ping @NicolasGensollen @KamalakerDadi let me suggest you collaborate on that one.

**Comment by KamalakerDadi** [comment]
We already have fetcher [there](https://github.com/Parietal-INRIA/DiFuMo/blob/master/notebook/fetcher.py) that is made for bulding a website. May be that could give a start. The atlases are stored on Open Science Framework and the fetcher gets them from there.

**Comment by GaelVaroquaux** [comment]
Yes, I think that now is a good time to integrate these fetcher to
nilearn.

**Comment by NicolasGensollen** [comment]
DiFuMo atlases were added in #2619 
Closing this...
